core/miner/storage_fsm/impl/events_impl.cpp

```diff
@@ -64,13 +59,9 @@ namespace fc::mining {
       OUTCOME_TRY(handler(tipset, best_height));
 
       lock.lock();
-      best_tipset = tipset_cache_->best();
+      OUTCOME_TRY(best_tipset, tipset_cache_->best());
```

Comment:
duplicate/shadowed variable `best_tipset`, is it intended?  
may use `OUTCOME_TRYA(best_tipset, ...)` to assign to existing variable.